### DISCUSSION TITLE
Feature Request: Replace local media URL with public domain

### DISCUSSION THREAD

**Discussion by colfin22** [CyferShepard/Jellystat#67] [Ideas]
When I click a link to open a library item in Jellyfin it's opening in a local URL. Please add the option to replace this URL to prefix with my public domain instead
http://jellyfin:8096/web/index.html#!/details?id=38bc83fd5e7b45a875bf789e106f2594
change to
https://jellyfin.mydomain.com/web/index.html#!/details?id=38bc83fd5e7b45a875bf789e106f2594

**Comment by CyferShepard** [comment]
Hi @colfin22 , this option is already available under settings, it's based on the Jellyfin URL field

**Reply by colfin22** [reply]
So it is, thank you

**Comment by colfin22** [comment]
I've done a wipe, installed 1.0.5 docker and this is no longer working for me. On initial login I set the url to be local hostname and port which connects fine to jellyfin but when I try to save my reverse proxied domain name in settings it does not save and Jellyfin links don't work
